### PR TITLE
[trello.com/c/yd3JjZ1f] feat: add chinese localization

### PR DESCRIPTION
**Pull request by StanislavDevIOS**

### CODE REVIEW

File: CommonKit/Sources/CommonKit/Assets/Localization/zh.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,1145 @@
...
+"Contribute.Section.CodeContributeDescription" = "如果您是开发人员，请加入我们的GitHub存储库";
+
+/* 贡献场景："捐赠"部分标题 */
+"Contribute.Section.Donate" = "捐赠";
+
+/* 贡献场景："捐赠"部分描述 */
+"Contribute.Section.DonateDescription" = "每一次捐赠都让ADAMANT变得更强大";
+
+/* 贡献场景："速率"部分标题 */
+"Contribute.Section.Rate" = "在app Store中对应用进行评分";
```
Comment (adamant-al) [inline CommonKit/Sources/CommonKit/Assets/Localization/zh.lproj/Localizable.strings:56]:
```suggestion
"Contribute.Section.Rate" = "在App Store中对应用进行评分";
```

File: CommonKit/Sources/CommonKit/Assets/Localization/zh.lproj/Localizable.strings [hunk 2]
```diff
@@ -0,0 +1,1145 @@
...
+"About.Row.Blog" = "我们的博客";
+
+/* 关于场景：推特排 */
+"About.Row.Twitter" = "推特";
+
+/* "关于场景：网站本地化url" */
+"About.Row.Website.Url" = "https://adamant.im";
+
+/* "关于场景：白皮书本地化url" */
+"About.Row.Whitepaper.Url" = "https://adamant.im/whitepaper/adamant-whitepaper-en.pdf";
```
Comment (adamant-al) [inline CommonKit/Sources/CommonKit/Assets/Localization/zh.lproj/Localizable.strings:83]:
```suggestion
"About.Row.Whitepaper.Url" = "https://adamant.im/whitepaper/adamant-whitepaper-cn.pdf";
```

File: CommonKit/Sources/CommonKit/Assets/Localization/zh.lproj/Localizable.strings [hunk 3]
```diff
@@ -0,0 +1,1145 @@
...
+"About.Row.GitHub.Url" = "https://github.com/Adamant-im";
+
+/* "关于场景：我们的博客本地化url" */
+"About.Row.Blog.Url" = "https://medium.com/adamant-im";
+
+/* "关于场景：Twitter本地化url" */
+"About.Row.Twitter.Url" = "https://twitter.com/adamant_im";
+
+/* 关于场景：写入Adamant行 */
+"About.Row.Adamant" = "Adamant";
```
Comment (adamant-al) [inline CommonKit/Sources/CommonKit/Assets/Localization/zh.lproj/Localizable.strings:95]:
```suggestion
"About.Row.Adamant" = "ADAMANT";
```

File: CommonKit/Sources/CommonKit/Assets/Localization/zh.lproj/Localizable.strings [hunk 4]
```diff
@@ -0,0 +1,1145 @@
...
+"AccountTab.Row.VoteForDelegates" = "为代表投票";
+
+/* "帐户"选项卡："操作"部分标题 */
+"AccountTab.Section.Actions" = "行动";
+
+/* "帐户"选项卡："应用程序"部分标题 */
+"AccountTab.Section.Application" = "应用";
+
+/* 账户选项卡：阿达曼钱包 */
+"AccountTab.Wallets.adamant_wallet" = "adamant钱包";
```
Comment (adamant-al) [inline CommonKit/Sources/CommonKit/Assets/Localization/zh.lproj/Localizable.strings:230]:
```suggestion
"AccountTab.Wallets.adamant_wallet" = "ADAMANT钱包";
```

File: CommonKit/Sources/CommonKit/Assets/Localization/zh.lproj/Localizable.strings [hunk 5]
```diff
@@ -0,0 +1,1145 @@
...
+
+/* 账户选项卡：以太坊钱包 */
+"AccountTab.Wallets.ethereum_wallet" = "以太坊钱包";
+
+/* 账户选项卡：Lisk钱包 */
+"AccountTab.Wallets.lisk_vallet" = "lisk钱包";
```
Comment (adamant-al) [inline CommonKit/Sources/CommonKit/Assets/Localization/zh.lproj/Localizable.strings:236]:
```suggestion
"AccountTab.Wallets.lisk_vallet" = "Lisk钱包";
```

File: CommonKit/Sources/CommonKit/Assets/Localization/zh.lproj/Localizable.strings [hunk 6]
```diff
@@ -0,0 +1,1145 @@
...
+"ChatsProvider.Validation.MessageTooLong" = "消息太长";
+
+/* 已知联系人：Adamant ICO消息。支持降价 */
+"Chats.IcoMessage" = "Null";
+
+/* 已知联系人：Adamant预ICO消息 */
+"Chats.PreIcoMessage" = "Null";
+
+/* 已知联系人：Adamant欢迎信息。支持降价 */
+"Chats.WelcomeMessage" = "ADAMANT是一个真正的区块链信使，独立于政府、企业甚至开发商。这是由于去中心化的网络基础设施，完全开源，由用户运行。这就是为什么每一个动作，包括发送消息或保存联系人姓名，都有0.001 ADM的网络费用。\n\nBlockchain提供了卓越的安全性和隐私，这是典型的P2P和c集中的信使。此外，区块链提供了新的可能性。您可以在聊天中存储和传输密码，并完全控制您的私钥，使用ADAMANT作为2FA等。\\n\nADAMANT中，没有人可以控制、阻止、停用、限制或审查帐户。用户对内容、消息、媒体以及使用Messenger的目标和意图承担全部责任。\\n\n请注意，您的安全性和匿名性取决于您。不要关注你收到的链接，否则你的IP可能会被泄露。在设备上设置密码。了解有关安全性和匿名性的更多信息，请访问https://adamant.im/staysecured/.\\n\n请确保已保存此帐户的密码短语--注销并再次登录。把你的通行证短语也写在纸上。只有**您对通行证短语的安全负责**。它无法恢复。如果其他人得到了它，你的代币就会被盗，信件也会被读取。把这个问题看得最重要，就好像你钱包里的代币在一段时间内会花费10亿美元一样。\\n\n若要立即开始发送消息，请在\"帐户\"选项卡上**获得免费欢迎令牌**。然后创建新聊天并输入您朋友的ADM地址。建议您亲自分享ADM地址，不要使用其他信使。";
```
Comment (adamant-al) [inline CommonKit/Sources/CommonKit/Assets/Localization/zh.lproj/Localizable.strings:329]:
```suggestion
"Chats.WelcomeMessage" = "ADAMANT是一个真正的区块链信使，独立于政府、企业甚至开发商。这是由于去中心化的网络基础设施，完全开源，由用户运行。这就是为什么每一个动作，包括发送消息或保存联系人姓名，都有0.001 ADM的网络费用。\n\nBlockchain提供了卓越的安全性和隐私，这是典型的P2P和c集中的信使。此外，区块链提供了新的可能性。您可以在聊天中存储和传输密码，并完全控制您的私钥，使用ADAMANT作为2FA等。\n\nADAMANT中，没有人可以控制、阻止、停用、限制或审查帐户。用户对内容、消息、媒体以及使用Messenger的目标和意图承担全部责任。\n\n请注意，您的安全性和匿名性取决于您。不要关注你收到的链接，否则你的IP可能会被泄露。在设备上设置密码。了解有关安全性和匿名性的更多信息，请访问https://adamant.im/staysecured/.\n\n请确保已保存此帐户的密码短语--注销并再次登录。把你的通行证短语也写在纸上。只有**您对通行证短语的安全负责**。它无法恢复。如果其他人得到了它，你的代币就会被盗，信件也会被读取。把这个问题看得最重要，就好像你钱包里的代币在一段时间内会花费10亿美元一样。\n\n若要立即开始发送消息，请在\"帐户\"选项卡上**获得免费欢迎令牌**。然后创建新聊天并输入您朋友的ADM地址。建议您亲自分享ADM地址，不要使用其他信使。";
```

File: CommonKit/Sources/CommonKit/Assets/Localization/zh.lproj/Localizable.strings [hunk 7]
```diff
@@ -0,0 +1,1145 @@
...
+
+/* 已知联系人：Adamant Exchenge消息。支持降价 */
+"Chats.Exchange.WellomeMessage" = "嗨！😊 我是你的交换机器人。我即时匿名操作。键入* */help**看看我能做些什么。我要求的费用比我的机器人同伴高，所以我建议你以有竞争力的费用建立自己的机器人。ℹ️ 你可以在ADAMANT博客上了解更多信息。我非常可靠，但如果您有任何问题，请在这里留言U6386412615727665758。";
```
Comment (adamant-al) [inline CommonKit/Sources/CommonKit/Assets/Localization/zh.lproj/Localizable.strings:332]:
```suggestion
"Chats.Exchange.WelcomeMessage" = "嗨！😊 我是你的交换机器人。我即时匿名操作。键入* */help**看看我能做些什么。我要求的费用比我的机器人同伴高，所以我建议你以有竞争力的费用建立自己的机器人。ℹ️ 你可以在ADAMANT博客上了解更多信息。我非常可靠，但如果您有任何问题，请在这里留言U6386412615727665758。";
```

File: CommonKit/Sources/CommonKit/Assets/Localization/zh.lproj/Localizable.strings [hunk 8]
```diff
@@ -0,0 +1,1145 @@
...
+
+/* 已知联系人：Adamant押注比特币消息。支持降价 */
+"Chats.BetOnBitcoin.WelcomeMessage" = "嗨！😊 我是匿名的，区块链验证的投注机器人。我接受BTC费率的投注，并向中奖者支付奖励。ℹ️ 在ADAMANT的博客上了解更多信息，或键入* */help**开始下注。通过U63864126157276665758提供支持。";
+
+/* 已知联系人：Adamant Adelina消息。支持降价 */
+"Chats.Adelina.WelcomeMessage" = "你好，我是Adelina。让我们愉快地交谈一下！我可以做很多事情，比如：\n\n-起草电子邮件或其他文章\n-编写Python代码\n-回答关于一组文档的问题\n-支持对话\n-一系列主题的导师\n-翻译语言\n-写诗\n-模拟电子游戏中的角色等等";
+
+/* 已知联系人：阿达曼捐赠欢迎信息。支持降价 */
+"Chats.Donate.WellomeMessage" = "感谢您的支持，我们开发了去中心化的ADAMANT Messenger——在此聊天中转移ADM、ETH和其他代币。有关其他捐赠钱包，请参阅consided.im。";
```
Comment (adamant-al) [inline CommonKit/Sources/CommonKit/Assets/Localization/zh.lproj/Localizable.strings:341]:
```suggestion
"Chats.Donate.WelcomeMessage" = "感谢您的支持，我们开发了去中心化的ADAMANT Messenger——在此聊天中转移ADM、ETH和其他代币。有关其他捐赠钱包，请参阅adamant.im。";
```

File: CommonKit/Sources/CommonKit/Assets/Localization/zh.lproj/Localizable.strings [hunk 9]
```diff
@@ -0,0 +1,1145 @@
...
+"NodesEditor.DeleteNodeButton" = "删除";
+
+/* NodesEditor:主机行占位符 */
+"NodesEditor.HostRow.Placeholder" = "IP或主机名";
+
+/* NodesEditor:无法生成URL警报 */
+"NodesEditor.FailedToBuildURL" = "无效主机";
+
+/* 安全性：通知模式说明。支持降价 */
+"SecurityPage.Row.Notifications.ModesDescription" = "####通知模式\n\n####禁用\n无通知。\n\n####后台提取\n您的设备会自动提取新消息。没有外部调用。提取由iOS启动，实际时间由操作系统根据电池电量、蜂窝网络、应用程序使用模式等许多因素确定，无法预测。可以是20分钟、6小时，甚至可能是一天。不过，您仍然可以打开应用程序并查看新消息。\n\n####推送\n ADAMANT通知服务发送到您的设备的通知。在消息发送并获得区块链批准后，您几乎会立即收到通知——延迟几秒钟。但此模式要求您的设备在服务的数据库中注册其设备令牌。设备令牌是安全的，在大多数情况下建议使用此选项。\\n\n您可以在ADAMANT的Github页面上阅读更多关于设备注册的信息。\n \n";
```
Comment (adamant-al) [inline CommonKit/Sources/CommonKit/Assets/Localization/zh.lproj/Localizable.strings:701]:
```suggestion
"SecurityPage.Row.Notifications.ModesDescription" = "####通知模式\n\n####禁用\n无通知。\n\n####后台提取\n您的设备会自动提取新消息。没有外部调用。提取由iOS启动，实际时间由操作系统根据电池电量、蜂窝网络、应用程序使用模式等许多因素确定，无法预测。可以是20分钟、6小时，甚至可能是一天。不过，您仍然可以打开应用程序并查看新消息。\n\n####推送\n ADAMANT通知服务发送到您的设备的通知。在消息发送并获得区块链批准后，您几乎会立即收到通知——延迟几秒钟。但此模式要求您的设备在服务的数据库中注册其设备令牌。设备令牌是安全的，在大多数情况下建议使用此选项。\n\n您可以在ADAMANT的GitHub页面上阅读更多关于设备注册的信息。\n\n";
```

File: CommonKit/Sources/CommonKit/Assets/Localization/zh.lproj/Localizable.strings [hunk 10]
```diff
@@ -0,0 +1,1145 @@
...
+"Notifications.Mode.NotificationsDisabled" = "已禁用";
+
+/* 通知：使用后台提取通知 */
+"Notifications.Mode.BackgroundFetch" = "后台获取";
+
+/* 通知：使用Apple推送通知 */
+"Notifications.Mode.ApplePush" = "推送";
+
+/* 安全：访问Github */
+"SecurityPage.Row.VisitGithub" = "访问Github";
```
Comment (adamant-al) [inline CommonKit/Sources/CommonKit/Assets/Localization/zh.lproj/Localizable.strings:725]:
```suggestion
"SecurityPage.Row.VisitGithub" = "访问GitHub";
```

File: CommonKit/Sources/CommonKit/Assets/Localization/zh.lproj/Localizable.strings [hunk 11]
```diff
@@ -0,0 +1,1145 @@
...
+"WelcomeScene.Description.Slide2" = "ADAMANT是您的消息以及聊天加密货币传输工具的**匿名性**和**安全性**。\n\n[为什么其他消息不安全？](https://medium.com/adamant-im/adamant-security-features-e7cc836ff52c)";
+
+/* 欢迎：幻灯片3说明 */
+"WelcomeScene.Description.Slide3" = "ADAMANT基础设施不属于任何人，是**去中心化的**。\n\n它由ADM代币支付的消息维护。新帐户可能会收到免费代币用于测试。";
+
+/* 欢迎：幻灯片4说明 */
+"WelcomeScene.Description.Slide4" = "无需任何要求即可单击**创建帐户**（无需电子邮件/电话/任何内容）并邀请您的朋友。\n\n为了真正的安全起见，请亲自**共享您的ADAMANT地址。";
+
+/* 欢迎：幻灯片5说明 */
+"WelcomeScene.Description.Slide5" = "项目的源代码[完全开放](https://github.com/Adamant-im/).该应用程序正在开发中，我们不断添加新功能。\\n\n如果您是开发人员，请加入我们的团队。";
```
Comment (adamant-al) [inline CommonKit/Sources/CommonKit/Assets/Localization/zh.lproj/Localizable.strings:1109]:
```suggestion
"WelcomeScene.Description.Slide5" = "项目的源代码[完全开放](https://github.com/Adamant-im/).该应用程序正在开发中，我们不断添加新功能。\n\n如果您是开发人员，请加入我们的团队。";
```

File: CommonKit/Sources/CommonKit/Assets/Localization/zh.lproj/Localizable.strings [hunk 12]
```diff
@@ -0,0 +1,1145 @@
...
+
+/* Welcome: Description accept */
+"WelcomeScene.Description.Accept" = "我接受";
+
+/* 欧拉 */
+"EULA.Title" = "服务条款";
+"EULA.Accept" = "接受";
+"EULA.Decline" = "拒绝";
+"EULA.Text" = "ADAMANT Messenger基于去中心化的区块链技术，因此：\n\n✓ ADAMANT Messenger不容忍令人反感的内容或辱骂用户\n✓ ADAMANT Messenger最初专注于通信的安全性和用户的隐私。\n✓ ADAMANT Messenger始终显示完全开源的代码，许可证为GPL-3.0。源代码可在GitHub:GitHub.com/stend-im上获得。\n✓ ADAMANT Messenger按\"原样\"提供，不提供任何担保。\n✓ 开发人员只是ADAMANT Messenger的贡献者和用户。\n✓ 开发人员不控制用户的任何操作。\n✓ 开发人员在ADAMANT Messenger中没有权限。\n✓ 开发人员不可能禁用或阻止ADAMANT Messenger中的任何帐户。\n✓ 开发人员不可能获得加密密钥或读取任何ADAMANT Messenger用户的通信或其他信息。\n✓ 如果丢失，开发人员不可能恢复对任何用户帐户的访问。用户有责任保护自己的帐户的安全。\n✓ 开发人员不对用户内容、消息、媒体以及使用ADAMANT Messenger的目标和意图负责。\n✓ 开发者不存储用户之间传输的任何信息（消息、多媒体），也不对其在区块链上/中的存储负责。\n✓ 开发者不可能访问，也不对连接到ADAMANT Messenger或通过ADAMANT信使发送的任何资产（包括但不限于：加密货币）负责。\n✓ ADAMANT Messenger利用元数据作为账户之间通信的事实，在其区块链中公开提供。\n✓ 开发者不与任何第三方或外部服务合作提供任何使用数据。\n✓ ADAMANT信使基础设施（区块链节点）属于其用户，由用户运行。\n✓ 开发人员不可能停用、停止或暂停ADAMANT Messenger。开发人员对ADAMANT Messenger的运行不提供任何保证。\n✓ 开发人员不承担与ADAMANT Messenger相关的可能风险、成本和故障。\n✓ ADAMANT Messenger的用户从此对其使用承担全部责任，包括但不限于：区块链和Messenger内应用的其他技术的合法性，使用匿名消息服务，以及其他管辖法律。\\n\n通过使用ADAMANT Messenger，您从此接受这些服务条款。";
```
Comment (adamant-al) [inline CommonKit/Sources/CommonKit/Assets/Localization/zh.lproj/Localizable.strings:1127]:
```suggestion
"EULA.Text" = "ADAMANT Messenger基于去中心化的区块链技术，因此：\n\n✓ ADAMANT Messenger不容忍令人反感的内容或辱骂用户\n✓ ADAMANT Messenger最初专注于通信的安全性和用户的隐私。\n✓ ADAMANT Messenger始终显示完全开源的代码，许可证为GPL-3.0。源代码可在GitHub:GitHub.com/stend-im上获得。\n✓ ADAMANT Messenger按\"原样\"提供，不提供任何担保。\n✓ 开发人员只是ADAMANT Messenger的贡献者和用户。\n✓ 开发人员不控制用户的任何操作。\n✓ 开发人员在ADAMANT Messenger中没有权限。\n✓ 开发人员不可能禁用或阻止ADAMANT Messenger中的任何帐户。\n✓ 开发人员不可能获得加密密钥或读取任何ADAMANT Messenger用户的通信或其他信息。\n✓ 如果丢失，开发人员不可能恢复对任何用户帐户的访问。用户有责任保护自己的帐户的安全。\n✓ 开发人员不对用户内容、消息、媒体以及使用ADAMANT Messenger的目标和意图负责。\n✓ 开发者不存储用户之间传输的任何信息（消息、多媒体），也不对其在区块链上/中的存储负责。\n✓ 开发者不可能访问，也不对连接到ADAMANT Messenger或通过ADAMANT信使发送的任何资产（包括但不限于：加密货币）负责。\n✓ ADAMANT Messenger利用元数据作为账户之间通信的事实，在其区块链中公开提供。\n✓ 开发者不与任何第三方或外部服务合作提供任何使用数据。\n✓ ADAMANT信使基础设施（区块链节点）属于其用户，由用户运行。\n✓ 开发人员不可能停用、停止或暂停ADAMANT Messenger。开发人员对ADAMANT Messenger的运行不提供任何保证。\n✓ 开发人员不承担与ADAMANT Messenger相关的可能风险、成本和故障。\n✓ ADAMANT Messenger的用户从此对其使用承担全部责任，包括但不限于：区块链和Messenger内应用的其他技术的合法性，使用匿名消息服务，以及其他管辖法律。\n\n通过使用ADAMANT Messenger，您从此接受这些服务条款。";
```

**Review by adamant-al — CHANGES_REQUESTED** [review]
- Next time check such mistakes yourself
- Now look through if in-message variables are fine
- Next time ask a translator don't translate comments in `/* .. */`

**Review by just-software-dev — APPROVED** [review]

**Review by adamant-al — APPROVED** [review]

**Pull request merged**